olla/include/openmm/Platform.h

```diff
@@ -45,23 +45,42 @@ class Kernel;
 class KernelFactory;
 
 /**
- * A Platform defines an implementation of all the kernels needed to perform some calculation.
- * More precisely, a Platform object acts as a registry for a set of KernelFactory
- * objects which together implement the kernels.  The Platform class, in turn, provides a
- * static registry of all available Platform objects.
+ * A Platform defines an implementation of all the kernels needed to perform
+ * some calculation. More precisely, a Platform object acts as a registry for a
+ * set of KernelFactory objects which together implement the kernels.  The
+ * Platform class, in turn, provides a static registry of all available
+ * Platform objects.
  *
- * To get a Platform object, call
+ * OpenMM ships with platforms that provide high-performance simulation on a CPU
+ * or GPU. They are described in more detail in the User Manual. Usually, the
+ * platform is chosen automatically when a Context is constructed.
  *
+ * There are three main ways to get a Platform object:
+ *
+ * 1. If the name of the Platform is known, call getPlatformByName():
+ * \verbatim embed:rst:leading-asterisk
+ *     .. code-block:: cpp
+ *
+ *        Platform& platform = Platform::getPlatformByName(name);
+ * \endverbatim
+ *
+ * 2. If a Context has already been constructed, call Context::getPlatform():
  * \verbatim embed:rst:leading-asterisk
- * .. code-block:: cpp
+ *     .. code-block:: cpp
+ *
+ *        Platform& platform = context.getPlatform();
+ * \endverbatim
  *
- *    Platform& platform = Platform::findPlatform(kernelNames);
+ * 3. If the required kernels are known, call findPlatform():
+ * \verbatim embed:rst:leading-asterisk
+ *     .. code-block:: cpp
  *
+ *        Platform& platform = Platform::findPlatform(kernelNames);
  * \endverbatim
  *
- * passing in the names of all kernels that will be required for the calculation you plan to perform.  It
- * will return the fastest available Platform which provides implementations of all the specified kernels.
- * You can then call createKernel() to construct particular kernels as needed.
+ * The latter two methods returns the fastest available Platform which provides
+ * implementations of all the specified kernels. You can then call createKernel()
+ * to construct particular kernels as needed.
```

Comment:
Method 2 returns whatever Platform the Context is using.  That might be the fastest available, or it might be one that the user explicitly specified when creating the Context.